homeassistant/components/mealie/services.py

```diff
@@ -70,6 +72,31 @@
     }
 )
 
+SERVICE_SET_MEALPLAN = "set_mealplan"
+SERVICE_SET_MEALPLAN_SCHEMA = vol.Any(
+    vol.Schema(
+        {
+            vol.Required(ATTR_CONFIG_ENTRY_ID): str,
+            vol.Required(ATTR_DATE): cv.date,
+            vol.Required(ATTR_ENTRY_TYPE): vol.In(
+                [x.lower() for x in MealplanEntryType]
+            ),
```

Comment:
Should this me a selector with translations?

Reply:
it is